packages/core/src/actions/clearSuggestedActions.js

```diff
@@ -0,0 +1,5 @@
+import setSuggestedActions from './setSuggestedActions';
```

Comment:
Instead of reusing `setSuggestedActions`, could you change it to a new action `WEB_CHAT/CLEAR_SUGGESTED_ACTIONS`. The reducer will need to be updated but should be pretty straightforward.

Reply:
Sigh 😂